src/ol/renderer/layerrenderer.js

```diff
@@ -151,7 +151,7 @@ ol.renderer.Layer.prototype.loadImage = function(image) {
     // listener (a noop if the listener was already registered)
     goog.asserts.assert(imageState == ol.ImageState.IDLE ||
         imageState == ol.ImageState.LOADING);
-    goog.events.listenOnce(image, goog.events.EventType.CHANGE,
+    goog.events.listen(image, goog.events.EventType.CHANGE,
```

Comment:
Otherwise the change to ```ol.ImageState.LOADING``` is not listened for
See the corresponding similar change here: https://github.com/tschaub/ol3/commit/bf35b40b11790cf3e6dcd693bd663a0cdba4016b